### PR TITLE
Simple grammar/clarity improvements

### PR DESCRIPTION
**Pull request by nettofarah**

### CODE REVIEW

File: introduction/A_overview.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Phoenix is a web development framework written in Elixir which implements the server-side MVC pattern. Many of its components and concepts will seem familiar to those of us with experience in other web frameworks like Ruby on Rails or Python's Django.
 
-Phoenix provides the best of both worlds, high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
+Phoenix provides the best of both worlds: high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
```
Comment (lancehalvorsen) [inline introduction/A_overview.md]: I agree that the comma is wrong here, but I'm not convinced that a colon is correct either.

What would you think about a dash? ". . . best of both worlds - high developer productivity  . . . "

File: introduction/A_overview.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ Phoenix is actually the top layer of a multi-layer system designed to be modular
 Phoenix is made up of a number of distinct parts, each with its own purpose and role to play in building a web application. We will cover them all in depth throughout these guides, but here's a quick breakdown.
 
 - The Endpoint
-  - handles all aspects of requests up until the point where the router take over
+  - handles all aspects of requests up until the point where the router takes over
```
Comment (lancehalvorsen) [inline introduction/A_overview.md]: :+1:

File: introduction/A_overview.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ Phoenix is actually the top layer of a multi-layer system designed to be modular
...
   - provides a core set of plugs to apply to all requests
   - dispatches requests into a designated router
 - The Router
-  - parses incoming requests and dispatches to the correct controller/action, passing parameters as needed
+  - parses incoming requests and dispatches them to the correct controller/action, passing parameters as needed
```
Comment (lancehalvorsen) [inline introduction/A_overview.md]: :+1:

File: introduction/A_overview.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
-Phoenix provides the best of both worlds, high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
+Phoenix provides the best of both worlds: high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
 
-If you are already familiar with Elixir, great! If not, there are a number of places to learn. The [Elixir guides](http://elixir-lang.org/getting_started/1.html) are a great place to start. We also have a list of helpful resources in the [Learning Elixir and Erlang Guide](http://www.phoenixframework.org/docs/learning-elixir-and-erlang).
+If you are already familiar with Elixir, great! If not, there are a number of places to learn from. The [Elixir guides](http://elixir-lang.org/getting_started/1.html) are a great place to start. We also have a list of helpful resources in the [Learning Elixir and Erlang Guide](http://www.phoenixframework.org/docs/learning-elixir-and-erlang).
```
Comment (lancehalvorsen) [inline introduction/A_overview.md]: Thinking out loud about this one. In this sentence, "it" is the implied object of the verb "learn". (If not, there are a number of places to learn it.) Explicitly adding the "it" is a little awkward. The preposition "from" doesn't play the same role in the sentence as the object.

For now, let's please leave the original as it is.

**Comment by lancehalvorsen** [comment]
@nettofarah thanks very much for this! 

I made a minor comment or two. If you could tweak those, rebase, and squash the commits, we'll be all set.

Thanks again.

File: introduction/A_overview.md (hunk 1 above)
Comment (nettofarah) [inline introduction/A_overview.md]: sure. I'm happy with a dash.

File: introduction/A_overview.md (hunk 4 above)
Comment (nettofarah) [inline introduction/A_overview.md]: Sounds good!

**Comment by nettofarah** [comment]
@lancehalvorsen: we should be good now.
Thanks

**Comment by lancehalvorsen** [comment]
Great. Thanks so much @nettofarah!

**Pull request merged**